### ISSUE TITLE
No such elliptic arc exists: Regression between 1.4.1 and 1.4.2

### ISSUE THREAD

**Issue by yaqwsx** [mathandy/svgpathtools#171]
I have the following simple script:

```.py
import svgpathtools
import sys

document = svgpathtools.Document(sys.argv[1])
paths = document.paths()
```

When I run it on SVG below:
- it works on 1.4.1
- it doesn't work on 1.4.2 and further versions

![test](https://user-images.githubusercontent.com/1590880/156329498-0dd1984a-03f7-44ef-ad92-2674b6015014.svg)
[Zipped version](https://github.com/mathandy/svgpathtools/files/8168190/test.zip)

The stack trace is:
```
 File "/home/xmrazek7/.local/lib/python3.8/site-packages/svgpathtools/document.py", line 261, in paths
    return flattened_paths(self.tree.getroot(), group_filter,
  File "/home/xmrazek7/.local/lib/python3.8/site-packages/svgpathtools/document.py", line 143, in flattened_paths
    path = transform(parse_path(converter(path_elem)), path_tf)
  File "/home/xmrazek7/.local/lib/python3.8/site-packages/svgpathtools/path.py", line 309, in transform
    return transform_segments_together(curve, transformation)
  File "/home/xmrazek7/.local/lib/python3.8/site-packages/svgpathtools/path.py", line 191, in transform_segments_together
    transformed_segs = [transformation(seg) for seg in path]
  File "/home/xmrazek7/.local/lib/python3.8/site-packages/svgpathtools/path.py", line 191, in <listcomp>
    transformed_segs = [transformation(seg) for seg in path]
  File "/home/xmrazek7/.local/lib/python3.8/site-packages/svgpathtools/path.py", line 308, in <lambda>
    transformation = lambda seg: transform(seg, tf)
  File "/home/xmrazek7/.local/lib/python3.8/site-packages/svgpathtools/path.py", line 339, in transform
    return Arc(new_start, radius=new_radius, rotation=curve.rotation + rot,
  File "/home/xmrazek7/.local/lib/python3.8/site-packages/svgpathtools/path.py", line 1448, in __init__
    self._parameterize()
  File "/home/xmrazek7/.local/lib/python3.8/site-packages/svgpathtools/path.py", line 1503, in _parameterize
    raise ValueError("No such elliptic arc exists.")
ValueError: No such elliptic arc exists.
```

**Comment by yaqwsx** [comment]
Hi, any update on this? I would like to help, however, I don't have enough inside this fairly large codebase to actually be able to trace down the root cause of wrongly computing `radius_check`.

**Comment by mathandy** [comment]
Thanks for reporting this @yaqwsx!
This is now fixed in v1.5.1.

**Comment by yaqwsx** [comment]
Thanks, it seems that it works nicely! I updated my tools to use the newest release.
